docs.html

```diff
@@ -201,6 +201,67 @@ <h2>dangerouslySetInnerHTML</h2>
         <p>A more readable alternative style:</p>
         <pre><code id="dangerouslySetInnerHTML2" class="language-tsx">loading...</code></pre>
 
+        <h2>Styled Components</h2>
+        <h3>Nano JSX withStyles</h3>
+        <p>
+          Nano JSX offers multiple ways to handle CSS-Styling for your Components using
+          <code class="is-code">withStyles()</code>:
+        </p>
+
+        <ul class="dot-list">
+          <li>
+            Define styles as <code class="is-code">string</code>, <code class="is-code">function</code> or
```

Comment:
The params of `withStyles` are inaccurate.
The correct type is defined here https://github.com/nanojsx/nano/blob/c1b0001b00b294de66965aefd7e0458b0f51475f/src/withStyles.ts#L14.

Reply:
Please tell the correct definition for the correct params.

Reply:
You can pass to `withStyles()` an array of (or separated by a comma):

- `string`
- object that has `toString()`
- function that returns a `string`

There are all of these possibilities because some style-loaders return different formats.

Reply:
Thanks, I'll update the PR.

Reply:
I updated the information on withStyles() params and made some small adjustments.